third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -682,19 +643,15 @@ void AXPlatformNodeWin::OnActiveComposition(
 
 void AXPlatformNodeWin::FireUiaTextEditTextChangedEvent(
     const gfx::Range& range,
-    const base::string16& active_composition_text,
+    const std::u16string& active_composition_text,
     bool is_composition_committed) {
-  if (!::switches::IsExperimentalAccessibilityPlatformUIAEnabled()) {
-    return;
-  }
-
   // This API is only supported from Win8.1 onwards
   // Check if the function pointer is valid or not
   using UiaRaiseTextEditTextChangedEventFunction = HRESULT(WINAPI*)(
       IRawElementProviderSimple*, TextEditChangeType, SAFEARRAY*);
   UiaRaiseTextEditTextChangedEventFunction text_edit_text_changed_func =
       reinterpret_cast<UiaRaiseTextEditTextChangedEventFunction>(
-          ::GetProcAddress(GetModuleHandle(L"uiautomationcore.dll"),
+          ::GetProcAddress(GetModuleHandleW(L"uiautomationcore.dll"),
```

Comment:
Consider extracting the DLL name as a `constexpr`. It looks like we don't currently use it elsewhere right now.

Interesting that the previous code was calling the generic wrapper rather than the W version, but hardcoded a wide string rather than using the `TEXT()` macro. Thanks for fixing.

Reply:
Done